src/components/SkeletonList.ts

```diff
@@ -0,0 +1,35 @@
+import { SKELETON_REPEAT_TIME } from '../constant/movie';
+
+export default class SkeletonList extends HTMLElement {
+  connectedCallback() {
+    this.render();
+    this.mountSkeletonItem();
+  }
+
+  render() {
+    this.innerHTML = `
+        <ul class="item-list">
+      </ul>
+        `;
+  }
+
+  mountSkeletonItem() {
+    const skeletonItem = /*html*/ `
+        <li>
+        <a href="#">
+          <div class="item-card">
+            <div class="item-thumbnail skeleton"></div>
+            <div class="item-title skeleton"></div>
+            <div class="item-score skeleton"></div>
+          </div>
+        </a>
+      </li>
+        `;
+
+    const $itemList = this.querySelector('.item-list');
+    if ($itemList instanceof HTMLUListElement)
+      $itemList.innerHTML = skeletonItem.repeat(SKELETON_REPEAT_TIME);
```

Comment:
타입 가드의 활용이 아주 좋아요! 여기서 공백 depth가 생기니 return 구문으로 없애보면 어떨까요?

```suggestion
    if (!($itemList instanceof HTMLUListElement)) return;
    $itemList.innerHTML = skeletonItem.repeat(SKELETON_REPEAT_TIME);
```